docs/workflow/building/coreclr/windows-instructions.md

```diff
@@ -0,0 +1,62 @@
+# Build CoreCLR on Windows <!-- omit in toc -->
+
+* [Environment](#environment)
+* [Build the Runtime](#build-the-runtime)
+  * [Using the Command Line](#using-the-command-line)
+    * [CoreCLR](#coreclr)
+      * [Cross-Compilation](#cross-compilation)
+    * [Core_Root](#core_root)
+    * [Native ARM64 (Experimental)](#native-arm64-experimental)
+  * [Using Visual Studio](#using-visual-studio)
+
+This guide will walk you through building CoreCLR on Windows.
+
+## Environment
+
+Ensure you have all of the prerequisites installed from the [Windows Requirements](/docs/workflow/requirements/windows-requirements.md).
+
+## Build the Runtime
+
+As the main development platform for .NET, Windows has more than one way to build the _dotnet/runtime_ repo. You can do it entirely through the command-line as you'd do in Linux and MacOS, or you can leverage the integration with Visual Studio that the repository provides. This document will go on about both ways so you can choose whichever fits your needs best, or even use a combination of both.
+
+### Using the Command Line
+
+This section will cover how to build the product using the command-line.
```

Comment:
This just explains how to build CoreCLR. It is a fraction of the product that you cannot do much interesting with.

Do we need this doc at all?

Reply:
That's the point of the doc, hence it's within the `coreclr` folder of the `building` section. There are also instructions for Linux and MacOS. Why wouldn't we add a Windows one? It's just confusing to not have all platforms explained.

Reply:
I do not think that the OS-specific details (e.g. using docker container, cross-building, etc.) on how to build should be CoreCLR, Mono or Libraries specific sections. They apply equally to whole repo. 

For example, cross-building just the CoreCLR gives you a bunch of binaries that you cannot do much interesting with.

I think that these files (windows-instructions.md/linux-instructions.md/....) should be moved up and generalized so that they apply to building the whole repo.

Reply:
Upon further analysis, I think moving the docs up and therefore updating them (since they would now be at the repo's scope, rather than CoreCLR scope), would make things more accessible. However, I've noticed that our current docs are referenced by other sectors like some comments in `run.py` and libraries docs. So, I think we should do that on a separate PR, since it's a more voluminous change.

Reply:
This PR will need to update those other docs and links. Otherwise, it is will regress the experience, e.g. as highlighted in https://github.com/dotnet/runtime/pull/75569#discussion_r973179456 .

Reply:
https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md has a link to a CoreCLR specific instruction with this change in the middle of the text that was not there before. It has a good change to put off-track somebody who going through this for the first time and is interested in working on libraries or Mono.

Reply:
Which made no sense at all to have a link for CoreCLR and a link for "others". We should have either a link to each component or none at all. The latter being adding more universal instructions rather than we have right now. I still don't see how this is more confusing than it was before, scattered all over the place.